### PR TITLE
made changes in the "TheProgrammingClubBetaComponent.js" file, and added a new file "style.css" to add hover states to the buttons and links

### PR DESCRIPTION
**Pull request by abhineetpandey10**

### CODE REVIEW

**Comment by abhineetpandey10** [comment]
https://user-images.githubusercontent.com/54185164/103991661-020ae480-51b9-11eb-9941-e58e13c60007.mp4

The webpage UI now contains hover states for the buttons and links.

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from "react"
+import ReactDOM from "react-dom"
+import TheProgrammingClubFinalComponent from ".React Components/TheProgrammingClubBetaComponent.js"
+function App()
```
Comment (Samy-33) [inline frontend/src/App.js:5]: leave a blank line between imports and function

**Review by Samy-33 — CHANGES_REQUESTED** [review]

**Comment by Samy-33** [comment]
@abhineetpandey10, how is it responsive when there is a horizontal scroll bar?

**Comment by abhineetpandey10** [comment]
I had misinterpreted "**Responsive**". Have edited the comment accordingly, and have applied the requested changes in the "App.js" file.

**Comment by Samy-33** [comment]
> I had misinterpreted "**Responsive**". Have edited the comment accordingly, and have applied the requested changes in the "App.js" file.

No problem. Please read below about how to write good commit messages:
https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/

**Comment by MananJethwani** [comment]
this PR needs to be closed sorry @abhineetpandey10 aksh has already made most of the component and that will me merged soon and that includes correct fonts responsive site and mobile view as well extremly sorry for wasting your time

**Pull request closed without merging**